test/unit/sql/insert.test.js

```diff
@@ -31,8 +31,37 @@ describe(Support.getTestDialectTeaser('SQL'), () => {
           postgres: 'INSERT INTO "users" ("user_name") VALUES (\'triggertest\') RETURNING *;',
           default: "INSERT INTO `users` (`user_name`) VALUES ('triggertest');"
         });
+
     });
 
+    it('with bind parameters', () => {
+      const User = Support.sequelize.define('user', {
+        username: {
+          type: DataTypes.STRING,
+          field: 'user'
+        },
+        name: {
+          type: DataTypes.STRING,
+          field: 'name'
+        }
+      }, {
+        timestamps: false
+      });
+
+      const values = {
+        user: 'bindtest',
+        name: Support.sequelize.fn('UPPER', 'bindtest')
+      };
+      const options = {
+        bindParams: true
+      };
+      expectsql(sql.insertQuery(User.tableName, values, User.rawAttributes, options),
+        {
+          mssql: 'INSERT INTO [users] ([user],[name]) VALUES ($1,UPPER(N\'bindtest\'));',
```

Comment:
Ideally I'd like to assert on bind parameters as well. Something along the lines of 

```
expectsql(sql..., {
 query: {
 ....
  }.
  bind: [...]
})
```

Reply:
Good call - I'll do some tweaks shortly